base/api.py

```
@@ -49,3 +90,45 @@ def get(self, request: Request, *args, **kwargs):
+                    ),
+                ],
+            ),
+            500: OpenApiResponse(description="Internal server error"),
+        }
+    )
+    def post(self, request: Request, *args, **kwargs):
+        APP_ID = settings.RECLAIM_APP_ID
+        APP_SECRET = settings.RECLAIM_APP_SECRET
+        PROVIDER_ID = settings.RECLAIM_TWITTER_PROVIDER_ID
+
+        platform = request.query_params.get("platform")
```

Ruff (0.11.9)

Local variable `platform` is assigned to but never used

Remove assignment to unused variable `platform`

(F841)